docs/Technologies.md

```diff
@@ -1,61 +1,97 @@
 
-### pa11y
+### GraphQL Code Generator
 
-`pa11y` is a tool for accessibility testing. For context, By default, pa11y uses the WCAG2AA standard.
+[GraphQl Code Generator](https://www.the-guild.dev/graphql/codegen/) compiles typescript files for our API giving us the benefit of typing and more confidence in the types of data in our requests.
 
-If you would like to run pa11y against individual urls or with custom config as part of local development, consider installing plain ol' [pa11y](https://github.com/pa11y/pa11y) `yarn global add pa11y`. This allows you to do things like `pa11y --runner axe --runner htmlcs --standard WCAG2AAA http://localhost:3000`.
+## Monitoring
 
-To adjust warning levels, ignore certain types of warnings, or create actions (such as button clicks or user login) that happens in test runs reference the [pa11y configuration docs](https://github.com/pa11y/pa11y#configuration).
+### New Relic
+
+New Relic is used for endpoint monitoring and to observe Open Telemetry data in deployed environments. Read more in [Monitoring](../README.md#monitoring).
 
-#### pa11y with cypress-audit
+### Open Telemetry
 
-We are running pa11y on every page of our form inside of our Cypress tests using the cypress-audit library. This allows us to sign in and fill out the form using Cypress, and only rely on pa11y to scan those pages. This is also one place in our app where we have implemented a ratchet in CI. The issues present when we started testing with pa11y have been ignored, allowing CI to ensure that we aren't adding new errors in the future. Tickets have been filed to address those existing issues.
+Open Telemetry (OTEL) is a ecosystem of tools for collecting data about an application and its services. It provides a standard way to instrument code and is vendor agnostic. We send our OTEL data to New Relic where it helps us assemble tracing of all of our requests and record errors.
 
 ### Jaeger
 
 [Jaeger](https://www.jaegertracing.io/) is used to observe Open Telemetry data in local dev. Read more in the [Jaeger docs](https://www.jaegertracing.io/docs/1.35/getting-started/) or visit port [16686](http://localhost:16686)) on local dev.
 
-### New Relic
+## React Web Application
 
-New Relic is used for endpoint monitoring and to observe Open Telemetry data in deployed environments. Read more in [Monitoring](../README.md#monitoring).
+### Create React App
+
+[Create React App](https://facebook.github.io/create-react-app) builds and configures our React client, including typescripts, styles, and static assets. See also [react-scripts](https://github.com/facebook/create-react-app/tree/master/packages/react-scripts).
```

Comment:
not sure what "typescripts" are

Reply:
gotcha. reworded a bit below lmk if that is still unclear. wanted to help indicate that this tool is doing the frotnend ts to js compilation as well as bundling, building, configuring the web app